e2e_tests/tests/experiment/experiment.py

```diff
@@ -404,7 +404,7 @@ def num_error_trials(experiment_id: int) -> int:
 
 def trial_logs(trial_id: int, follow: bool = False) -> List[str]:
     return [
-        tl.message
+        str(tl.log) if tl.log is not None else ""
```

Comment:
you fill need to fmt the old message, i think, for now

Reply:
Yep! I created a `format_trial_log` that follows the formatting that had been applied previously! I just got it pushed up 😄